moveit_configs_utils/setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+package_name = "moveit_configs_utils"
+
+setup(
+    name=package_name,
+    version="0.0.0",
```

Comment:
Can we make the initial version 0.0.1? For some reason in MoveIt 1, [we used 0.3.0](https://github.com/ros-planning/moveit2/blob/main/moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template). Either way, 0.0.0 feels wrong to me.